templates/download/index.twig

```diff
@@ -67,6 +67,7 @@
 				<span class="fas fa-exclamation-circle"></span>
 				{% trans %}Downloads labeled Alpha, Beta, or Nightly are pre-release software, stability may suffer.{% endtrans %}
 			</small>
+			<br>Note: 32 bit nightly builds no longer available.
```

Comment:
```suggestion
			<br>{% trans %}Note: 32 bit nightly builds no longer available.{% endtrans %}
```

Reply:
I'm commiting this for now, but I don't understand how translations work. Does this automatically generate the correct translations or should I manually add entries?

Reply:
No idea but transiflex tracks them and they so it'll default to English just like LMMS does if missing.